docs/en/Configuring-Alluxio-with-secure-HDFS.md

```diff
@@ -62,6 +62,13 @@ Then edit `alluxio-site.properties` file to set the under storage address to the
 Copy secure HDFS conf xml files (`core-site.xml`, `hdfs-site.xml`, `mapred-site.xml`, `yarn-site.xml`) to
 `${ALLUXIO_HOME}/conf/`
 
+Optionally, you can set jvm-level system properties for customized Kerberos configurations:
+`java.security.krb5.realm` and `java.security.krb5.kdc`. If both are set to empty, Kerberos library will respect
+the default machine Kerberos configuration. For example, if you use Hadoop or Spark, you can set those two configurations in `HADOOP_OPTS` or `SPARK_OPTS` with `-D`.
+If you use Alluxio shell, you can add it to `ALLUXIO_JAVA_OPTS` in `conf/alluxio-env.sh`.
+
+{% include Configuring-Alluxio-with-secure-HDFS/alluxio-opts.md %}
```

Comment:
Well, I think we should keep a consistent style at least within the same file. Can we leave it as it is now `_include`, later on we will do a massive refactoring to make everything inline.